tests/test_lightningd.py

```diff
@@ -475,6 +475,23 @@ def test_invoice_weirdstring(self):
         b11 = l1.rpc.decodepay(inv['bolt11'])
         assert b11['description'] == weird_desc
 
+        # We can also use numbers as labels.
+        weird_label = 25
+        weird_desc = '"'
+        l1.rpc.invoice(123000, weird_label, weird_desc)
+        # FIXME: invoice RPC should return label!
+
+        # Can find by this label.
+        inv = l1.rpc.listinvoices(weird_label)['invoices'][0]
```

Comment:
We should probably `delinvoice` the invoice from earlier in the same test, as otherwise index 0 of the invoices list is likely to be that one rather than the one we just created.

Reply:
Good idea; we didn't test delinvoice with weird labels and this is a good opportunity too.